hack/verify-govet.sh

```diff
@@ -56,7 +56,8 @@ DIR_BLACKLIST='./hack
 ./pkg/user
 ./pkg/util
 ./test
-./tools'
+./tools
+'
```

Comment:
This is going to keep other delete diffs clean if they touch the end of the list